hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/error/ErrorBuilder.java

```diff
@@ -10,78 +12,204 @@
 	
 	private static String ack= "ACK";
 	
-	private static String version = "2.4\n";
+	private static String msh= "MSH";
 
 	private static String msa = "MSA|AR|";
-
-	private static String seperator3 = "|||";
+	
+	private static String unknown = "UNKNOWN";
+	
+	private static String sendingOrReceivingApp = "HNCLIENT";
+	
+	private  static String double_backslash = "\\"; // For using specific string in regex mathces
+	
+	private  static String delimiter = "|";
 
 	private static Timestamp timestamp = new Timestamp(System.currentTimeMillis());
 
-	private static String processingDomain = "D";
-
 	private static String sendingFacility;
 
-	private static String recievingFacility;
-
+	private static String receivingFacility;
```

Comment:
sendingFacility and receivingFacility are static variables which are essentially global and could be modified by anyone accessing the static methods at any given time, meaning they are not thread safe. I think it's a simple as making these local variables in the methods which use them.